translations/pt/get5.phrases.txt

```diff
@@ -6,11 +6,11 @@
     }
     "WaitingForCastersReadyInfoMessage"
     {
-        "pt"            "Esperando pelos streammers para digitar {2} pra começar."
+        "pt"            "Esperando pelos streamers digitar {2} pra começar."
```

Comment:
Are you sure the right word is "streamers" here? It's the spectators/commentators, also called "casters", that must be ready. @SidiBecker

Reply:
Yeah, that's wrong. It should be use {1} param there. I'll fix.

Reply:
Also that.

Reply:
No, sorry. It's `{2}` because the first argument is the name of the team, i.e. "Casters", for the casters. But since you defined the casters simply as "streamers", without the name they were given in the match, the argument is not used.

Reply:
`"Aguardando {1} digitar {2} pra começar."` would be right?

Reply:
I would assume, yes, with the reservation that I don't speak Portuguese.

Reply:
So yes, its like `"Waiting for {1} to type {2} to begin."`. I'll fix.